### DISCUSSION TITLE
Feature Request specifically send photos or videos from a particular group

### DISCUSSION THREAD

**Discussion by ThatDevCoder** [aahnik/tgcf#116] [General]
First of all, thank you for the script!
Can you please create a feature that can specifically send photos or videos from a particular group? Because there is a lot of unnecessary stuff also forwarded from the group which we are forwarding.

Hands down, best working script on Github!

**Comment by aahnik** [comment]
Filtering messages by file type is coming soon!

Please spread the word about `tgcf` by sharing on social media. And also don't forget to star the repo.

**Comment by aahnik** [comment]
this feature is now availaible! use version 0.1.27 or above.
